.github/ISSUE_TEMPLATE/doc-issue.md

```diff
@@ -8,12 +8,9 @@ about: Create an issue to help us improve
 If the issue is with an ASP.NET Core document:
 
 * Do **not** open a new issue using this form.
-* Open the issue with the **This page** button and form at the bottom of the document's page.
+* Open the issue using the **Open a documentation issue** link and feedback form at the bottom of the article. 
```

Comment:
```suggestion
* Open the issue using the **Open a documentation issue** link and feedback form at the bottom of the article:
<img  alt="new issues" src="https://github.com/dotnet/AspNetCore.Docs/assets/3605364/13d3fae0-7960-4233-80e7-02fdaa65e61a">
```

Reply:
It won't show up in the GH issue UI. Text only.